devicemodel/hw/pci/passthrough.c

```diff
@@ -599,6 +605,22 @@ passthru_deinit(struct vmctx *ctx, struct pci_vdev *dev, char *opts)
 		 */
 		vm_deassign_pcidev(ctx, &pcidev);
 	}
+	if (phys_bdf) {
```

Comment:
We should move this reset to !rtvm path. We don't deassign pcidev if the guest is RTVM. Otherwise, we break the feature that acrn-dm exit shouldn't impact RTVM.

Reply:
Make sense. It will be updated.